src/woo_publications/publications/admin.py

```diff
@@ -28,10 +29,11 @@ class DocumentInlineAdmin(admin.StackedInline):
 @admin.register(Publication)
 class PublicationAdmin(AdminAuditLogMixin, admin.ModelAdmin):
     list_display = (
+        "uuid",
```

Comment:
please put this right before `show_actions` as it was - the documentation now describes that the title is clickable in the admin, this changes that behaviour

The UUID also holds zero meaningful information to humans, it's important for technical reasons and we want to easily be able to copy-paste it (from the list page), but it definitely should not be the first column